README.md

```diff
@@ -2,9 +2,7 @@
 [![Paper shield](https://img.shields.io/badge/JMLR-Paper-red)](http://www.jmlr.org/papers/volume20/19-490/19-490.pdf)
 [![PyPI version](https://img.shields.io/pypi/v/graspologic.svg)](https://pypi.org/project/graspologic/)
 [![Downloads shield](https://pepy.tech/badge/graspologic)](https://pepy.tech/project/graspologic)
-[![Docs shield](https://img.shields.io/readthedocs/graspologic)](https://graspologic.readthedocs.io/)
```

Comment:
no longer applies if we aren't using RTD

Reply:
Good catch.